theme/assets/src/block-editor/blocks/back-to-top/block.json

```diff
@@ -10,9 +10,6 @@
     "align": true
   },
   "description": "Material back to top button",
-  "editorStyle": [
-    "material-google-fonts",
-    "material-block-editor-css-theme"
-  ],
+  "editorStyle": "material-google-fonts",
```

Comment:
Kept this style to font since we want to load material font for icons in editor.